src/drivers/windows/lv_windows_display.c

```diff
@@ -111,10 +111,74 @@ int32_t lv_windows_dpi_to_physical(int32_t logical, int32_t dpi)
     return MulDiv(logical, dpi, USER_DEFAULT_SCREEN_DPI);
 }
 
+void lv_windows_set_top_level(lv_display_t * display, bool top_level)
+{
+    HWND window_handle = lv_windows_get_display_window_handle(display);
+
+    LONG_PTR ex_style = GetWindowLongPtr(window_handle, GWL_EXSTYLE);
+    if(top_level) {
+        ex_style |= WS_EX_TOPMOST;
+        ex_style |= WS_EX_APPWINDOW;
+    }
+    else {
+        ex_style &= ~WS_EX_TOPMOST;
+    }
+
+    SetWindowLongPtr(window_handle, GWL_EXSTYLE, ex_style);
+    SetWindowPos(window_handle, top_level ? HWND_TOPMOST : HWND_NOTOPMOST, 0, 0, 0, 0,
```

Comment:
We should type best to keep one parameter for each line when we calling the function to improve the code readability. You also should keep maximum 80 half-wide characters per line.

For example, learn the coding style when I calling CreateWindowEx in lv_windows_display_thread_entrypoint function.
```
    HWND window_handle = CreateWindowExW(
                             WS_EX_CLIENTEDGE,
                             L"LVGL.Window",
                             data->title,
                             window_style,
                             CW_USEDEFAULT,
                             0,
                             data->hor_res,
                             data->ver_res,
                             NULL,
                             NULL,
                             NULL,
                             data);
```

There are some similar code snippets you can improve.